content/en/docs/reference/kubernetes-api/authentication-resources/certificate-signing-request-v1.md

```diff
@@ -239,7 +242,8 @@ CertificateSigningRequestStatus contains conditions used to indicate approved/de
 
 ## CertificateSigningRequestList {#CertificateSigningRequestList}
 
-CertificateSigningRequestList is a collection of CertificateSigningRequest objects
+\<p>CertificateSigningRequestList is a collection of CertificateSigningRequest objects\</p>
```

Comment:
The way this has rendered doesn't look right to me.

Reply:
This is a dilemma. We wanted to treat source comment as markdown because most of them are following markdown syntax. However, when we preprocess the comment via the markdown package, there is always a `<p>`/`</p>` pair in the rendered output.
Let's see how the result looks like.

Reply:
Okay, looks like the backslash are not good. Will investigate it.

Reply:
If we can take the Markdown input and yield Markdown (no HTML yet), we can hope that Hugo does the rest.

What we can do in the generator wrap the chunk of Markdown-from-OpenAPI in a shortcode of our choice. The shortcode doesn't need to do much but it will force Hugo to close any unclosed HTML elements and we can also (eg) set a class on the outer element to help with styling.

How about that?

Reply:
It was on the tip of my tongue, what's the reason to use html inside markdown outputs?

Reply:
Need to check the generator source (used to be maintained by someone else).